docs: update fast mode diagrams (#748)
yetanotherco/aligned_layer: docs/architecture/1_fast_mode.md

## Fast mode in a nutshell

The fast mode works using a subset of Ethereum’s validators via restaking.
Validators (also called Operators) receive proofs, verify them using the verification code written in Rust or another
higher-level language, and sign messages with BLS signatures.
If a two-thirds majority agrees, the results are posted to Ethereum.

## Simplified Architecture

The proof submission can be simplified as follows:

![Figure 1: Simplified Architecture.png](../images/simplified_architecture.png)

1. The users submit proofs to the batcher through the CLI or SDK.

> 2. The batcher accumulates proofs, and then sends the batch to the Data Service and posts the merkle root and data to

Reviewer: ```suggestion
2. The batcher accumulates proofs, and then sends the batch to some Data Service and posts the merkle root and data to
```

8. The signature aggregator accumulates the signed responses until reaching the quorum, then sends the aggregated
   signature to Ethereum.
9. Ethereum verifies the aggregated signatures and changes the state of the batch from pending to verified.

> ### Batch structure
> The task batch consists of a Merkle tree containing the relevant information for proof verification in the lower level leaves. The root of the Merkle tree is posted to Ethereum together with a pointer to where the data is stored. Each leaf contains the following information:

Reviewer 2: ```suggestion
### Batch structure

The task batch consists of a Merkle tree containing the relevant information for proof verification in the lower level leaves. The root of the Merkle tree is posted to Ethereum together with a pointer to where the data is being stored. Each leaf contains the following information:
```

- A commitment to the public input of the proof.
- A commitment to the proof and information about the proof system.
- A commitment to the program or a commitment to the verification key (depending on the proof system used).
- The address of the proof’s generator/submitter (optional).

A diagram for the batch is shown on the figure below:

![Figure 3: Structure of a batch](../images/batch.png)

> ### Reading the results from Ethereum
> Once the results from a batch have been checked on Ethereum, the Aligned contract is updated with the results. The user’s contract can query the Aligned contract to check whether the proof has been included in a successful batch.

Reviewer 2: ```suggestion
### Reading the results from Ethereum

Once the results from a batch have been checked on Ethereum, the Aligned contract is updated with the results. The user’s contract can query the Aligned contract to check whether the proof has been included or not in a verified batch.
```
